tool/src/webapp/js/show-page.js

```diff
@@ -1157,7 +1173,33 @@ $(function() {
 			$("#grouplist").hide();
 			return false;
 		});
-		
+		//when edit twitter link is clicked twitterDialog is opened
+		$(".edit-twitter").click(function(){
+			closeDropdowns();
+			var row = $(this).parent().parent().parent();
```

Comment:
Can stay in but this is brittle to changes in the HTML can easily break this. Much better to be more generic and more specific.